wordpress-mobile/WordPress-iOS
# History / Revisions: HTML render with Aztec

Refs. #10304 

![revisions-visual-preview-edit](https://user-images.githubusercontent.com/912252/50167818-e8508900-02e1-11e9-8856-fe79669399e2.gif)

This PR implements the last task about the History / Revisions V2. It adds the possibility to switch between the visual and HTML view.

It basically is a continuation of what @nheagy already did with #10468 , using _Aztec_ to render the HTML content, adding a `RevisionPreviewTextViewManager` to manage the `TextViewAttachmentDelegate`. It also uses the same approach of the _Post Editor_ to display the title and the content:
- Label for the title
- Aztec TextView for the content

**To test:**
- Select a Revision from the revisions list
- Click on ••• button (top right of the navigation bar) and select _Switch to Visual Preview_
- You should see the HTML rendered by _Aztec_ view

Update release notes:


@jtreanor can you have a look to this? ☝️
@diegoreymendez you are the Aztec expert! Any suggestion is really welcomed! 😊
